backend/pyproject.toml

```diff
@@ -7,17 +7,16 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.12"
-fastapi = {extras = ["standard"], version = "^0.112.2"}
-sqlmodel = "^0.0.21"
-psycopg2-binary = "^2.9.9"
-alembic = "^1.13.2"
-pytest-mock = "^3.14.0"
+fastapi = {extras = ["standard"], version = "^0.115.8"}
+psycopg2-binary = "^2.9.10"
```

Comment:
let's change to psycopg3?

Reply:
Dockerfile will need to install `libpq-dev` in the build phase.

`poetry add "psycopg[pool]"`

```
DATABASES = {
    "default": {
        "ENGINE": "django.db.backends.postgresql",
        # ...
        "OPTIONS": {
            "pool": {
                "min_size": 2,
                "max_size": 4,
                "timeout": 10,
            }
        },
    },
}
```